modules/react/tabs/lib/useTabsModel.tsx

```diff
@@ -8,6 +8,22 @@ import {
 
 import {useMenuModel} from '@workday/canvas-kit-react/menu';
 
+/**
+ * The TabsModel extends the [Collection
+ * System](/getting-started/for-developers/resources/collection-api/). Tabs have tab items and
+ * panels. Tabs can be overflowed and a MenuModel sub-model.
```

Comment:
This doesn't read right, I wonder if this is better:
```suggestion
 * panels. Tabs can be overflowed and use a MenuModel sub-model.
```

Reply:
I think I meant "to a" instead of "and a"